src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1021,9 +1027,13 @@ public virtual void Save(string filePath, bool isBackup = false, EngineControlle
 
                 // Stage 1: Serialize the workspace.
                 var json = this.ToJson(engine);
```

Comment:
@mjkkirschner @scottmitchell I wonder if you guys have discussed adding a Json Serializable property `Extensions` with getter only so that ToJson() will take care of it by default. I believe 3rd party developer should not need to know special workaround `AddExtensionBlockToJSON` to be able to save the Model into the official scheme we have. Otherwise, when they make the call and for example upload the json to us, it will be missing information. My 2 cents

Reply:
That makes sense. I'll take a look at this.